client/src/components/Main/RSVPFloating/index.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { theme1, grey3, lightBlue, eventPink, eventGreen } from "../../../resources/style-constants";
+import SecondaryButton from "../../Common/Buttons/SecondaryButton/index.js";
+import PrimaryButton from "../../Common/Buttons/PrimaryButton/index.js";
+import RSVPCard from "../RSVPCard/index.js"
+import styled from "styled-components";
+
+const accentTheme = {
+  default: lightBlue,
+  hover: theme1,
+  disabled: grey3,
+  borderRadius: 0
+};
+
+const FormWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  padding: 0px;
+  margin: 5px;
+  |
+`;
+const AlightRight = styled.div`
+display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    width: 45%;
+    height: 100%;
+    position: absolute;
```

Comment:
Personally, I prefer not to use position: absolute very often. One way around this is that in the parent component, you can use justify-content: space-between.